src/main/java/core/basesyntax/figures/Figure.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.utils.CalculateArea;
+import core.basesyntax.utils.Color;
+import core.basesyntax.utils.DrawFigure;
+
+public abstract class Figure implements DrawFigure, CalculateArea {
+    private double area;
+    private Color color;
```

Comment:
better to do it protected, as we want to access it in child classes
```suggestion
    protected Color color;
```

Reply:
not fixed